geode-core/src/main/java/org/apache/geode/internal/cache/TXManagerImpl.java

```diff
@@ -680,37 +695,85 @@ private void closeListener(TransactionListener tl) {
    *         to reactivate the suspended transaction.
    */
   public TXStateProxy internalSuspend() {
```

Comment:
Is this method now only used by tests? If so say so in the comment. I would recommend deprecating it and say you should use pauseTransaction of the external "suspend" instead. This will keep new code from accidently using this method.